### ISSUE TITLE
P2: Provide clear description of memory at any timeslice (Dart, Native, Graphic, Skia, etc.)

### ISSUE THREAD

**Issue by kenzieschmoll** [flutter/devtools#2538]
Allow the developer to [dive into the memory usage as appropriate](https://github.com/flutter/flutter/issues/56486#issuecomment-637166071) showing hot spots and where those allocations came from in their code.

The developer should be able to find leaks or spots in their code that cause large memory allocations, and see whether that memory is Dart, Native, Graphic, Skia, etc.

**Comment by kenzieschmoll** [comment]
We are waiting on some additional information to be exposed from the engine.

**Comment by xster** [comment]
Is there a bug tracking the additional information needed from the engine? Is the engine team aware of this request? @zanderso

**Comment by zanderso** [comment]
This was a while back, so if there were issues filed in the Flutter issue tracker for the specific items, I suspect they've fallen through the cracks and will need to be pinged.

**Comment by kenzieschmoll** [comment]
This issue is a stale request from 2020 that depends on additional information being exposed by the Flutter engine, which was not followed up on. If there is more work to do here, please let us know by filing a new issue with up to date information. Thanks!
